### PR TITLE
Homework6: Ekaterina Zavadskaya

### PR DESCRIPTION
**Pull request by KateZavadskaya**
Add homework6

Add 5 files: hm6_1.py - hm6_5.py

Docs: hm6_1.py - hm6_5.py

### CODE REVIEW

**Comment by KateZavadskaya** [comment]
@Stml89 Can you check my homework6, please.

File: hm6_1.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+""" Task 1: Перевести строку в список
+
+"Robin Singh" => ["Robin”, “Singh"]
+
+"""
+
+TXT = "Robin Singh"
+
+print(TXT, type(TXT))
+print(TXT.split(), type(TXT.split()))
```
Comment (Stml89) [inline hm6_1.py]: the same issue as in HW5

File: hm6_2.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+""" Task 2: "I love arrays they are my favorite"
+
+=> ["I", "love", "arrays", "they", "are", "my", "favorite"]
+
+"""
+
+TXT = "I love arrays they are my favorite"
+
+print(TXT, type(TXT))
+print(TXT.split(), type(TXT.split()))
```
Comment (Stml89) [inline hm6_2.py]: here as well

**Review by Stml89 — CHANGES_REQUESTED** [review]
small comment to fix

File: hm6_1.py (hunk 1 above)
Comment (KateZavadskaya) [reply hm6_1.py]: Understand your message, will fix, thank you.

File: hm6_2.py (hunk 2 above)
Comment (KateZavadskaya) [reply hm6_2.py]: Understand your message, will fix, thank you.

**Comment by KateZavadskaya** [comment]
@Stml89 fixed your comments, check it 🤞, please.

**Review by Stml89 — APPROVED** [review]

**Pull request merged**